conf/aggregations_test.go

```diff
@@ -91,31 +91,48 @@ func TestReadAggregations(t *testing.T) {
 		},
 		{
 			title: "lots of comments",
-			in: `;[this is not a section]
+			in: `#[this is not a section]
 			[foo] # [commented]
 			pattern = fo#ooo;.* # another comment here. note that literal ; and # are allowed
-			; pattern = this-should-be-ignored
+			# pattern = this-should-be-ignored
 			xFilesFactor = 0.8 # comment
-			;xFilesFactor = 0.9
-			;aggregationMethod = min,avg
+			#xFilesFactor = 0.9
+			#aggregationMethod = min,avg
 			#aggregationMethod = min,avg
 			aggregationMethod = max
-			;aggregationMethod = min,avg
 			#aggregationMethod = min,avg
-			; and a final comment on its own line`,
+			# and a final comment on its own line`,
 			expErr: false,
 			expAgg: Aggregations{
 				Data: []Aggregation{
 					{
 						Name:              "foo",
-						Pattern:           regexp.MustCompile("fo#ooo;.*"),
+						Pattern:           regexp.MustCompile("fo"),
 						XFilesFactor:      0.8,
 						AggregationMethod: []Method{Max},
 					},
 				},
 				DefaultAggregation: defaultAggregation(),
 			},
 		},
+		{
+			title: "pattern with special characters",
+			in: `[foo]
+			pattern = [^ab](x|y)\p{Greek}.*,:;!@%&=+-_/?<>"'~$#ab
```

Comment:
This looks a good test for the issue I had.

Reply:
the configparser PR actually has a new unit test with your exact pattern, btw :)